extensions/amp-story/1.0/amp-story.js

```diff
@@ -545,6 +547,10 @@ export class AmpStory extends AMP.BaseElement {
       this.onPausedStateUpdate_(isPaused);
     });
 
+    this.storeService_.subscribe(StateProperty.SIDEBAR_STATE, openSidebar => {
```

Comment:
Nit: s/openSidebar/sidebarState (or any name you like that makes it obvious it's a boolean)